### ISSUE TITLE
Compiler option "enableIvy": false causes app to crash with StaticInjectorError message

### ISSUE THREAD

**Issue by jindrichruzicka** [angular/flex-layout#1189]
# Bug Report

#### What is the expected behavior?
App using FlexLayoutModule should be able to run with disabled Ivy option.

#### What is the current behavior?
App crashes when used with "enableIvy": false compiler option and together with FlexLayoutModule.

The error displayed in console reads:
ERROR NullInjectorError: StaticInjectorError(AppModule)[FlexFillDirective -> StyleUtils]: 

#### What are the steps to reproduce?
Disable Ivy using tsconfig.json file, for example like this:

```
"angularCompilerOptions": {
    "enableIvy": false
  }
```

Import FlexLayoutModule in yout app and just use it anywhere, for example like this:

```
<div fxLayout="row" fxFlexFill>
  <div>test</div>
  <div>test</div>
</div>
```

#### What is the use-case or motivation for changing an existing behavior?
We need to make this work in our project as we are not ready to adopt Ivy yet and we are migrating to new version of Angular.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Tested on 
Angular 0.900.0-rc.11
Material 9.0.0-rc.8
OS Windows 10
Typescript 3.7.5
Browser Chrome (version should not matter)

#### Is there anything else we should know?
If you mix eggs with a ham, you can make yourself nice ham and eggs.

The Error
```
main.js:277 NullInjectorError: NullInjectorError: No provider for StyleUtils!
    at NullInjector.get (http://localhost:4200/vendor.js:8285:27)
    at resolveToken (http://localhost:4200/vendor.js:22746:24)
    at tryResolveToken (http://localhost:4200/vendor.js:22672:16)
    at StaticInjector.get (http://localhost:4200/vendor.js:22513:20)
    at resolveToken (http://localhost:4200/vendor.js:22746:24)
    at tryResolveToken (http://localhost:4200/vendor.js:22672:16)
    at StaticInjector.get (http://localhost:4200/vendor.js:22513:20)
    at resolveNgModuleDep (http://localhost:4200/vendor.js:35017:29)
    at NgModuleRef_.get (http://localhost:4200/vendor.js:35877:16)
    at resolveDep (http://localhost:4200/vendor.js:36402:45)
    at createClass (http://localhost:4200/vendor.js:36260:32)
    at createDirectiveInstance (http://localhost:4200/vendor.js:36071:22)
    at createViewNodes (http://localhost:4200/vendor.js:47572:38)
    at callViewAction (http://localhost:4200/vendor.js:48022:13)
    at execComponentViewsAction (http://localhost:4200/vendor.js:47927:13)
    at createViewNodes (http://localhost:4200/vendor.js:47601:5)
    at createRootView (http://localhost:4200/vendor.js:47444:5)
    at callWithDebugContext (http://localhost:4200/vendor.js:48975:27)
    at Object.debugCreateRootView [as createRootView] (http://localhost:4200/vendor.js:48211:12)
    at ComponentFactory_.create (http://localhost:4200/vendor.js:35416:31)
    at ComponentFactoryBoundToModule.create (http://localhost:4200/vendor.js:31863:29)
    at ApplicationRef.bootstrap (http://localhost:4200/vendor.js:44496:42)
    at http://localhost:4200/vendor.js:44122:25
    at Array.forEach (<anonymous>)
    at PlatformRef._moduleDoBootstrap (http://localhost:4200/vendor.js:44118:44)
    at http://localhost:4200/vendor.js:44073:26
    at ZoneDelegate.invoke (http://localhost:4200/polyfills.js:378:30)
    at Object.onInvoke (http://localhost:4200/vendor.js:43260:33)
    at ZoneDelegate.invoke (http://localhost:4200/polyfills.js:377:36)
    at Zone.run (http://localhost:4200/polyfills.js:137:47)
    at http://localhost:4200/polyfills.js:864:40
    at ZoneDelegate.invokeTask (http://localhost:4200/polyfills.js:413:35)
    at Object.onInvokeTask (http://localhost:4200/vendor.js:43238:33)
    at ZoneDelegate.invokeTask (http://localhost:4200/polyfills.js:412:40)
    at Zone.runTask (http://localhost:4200/polyfills.js:181:51)
    at drainMicroTaskQueue (http://localhost:4200/polyfills.js:583:39)
```

**Comment by mhevery** [comment]
https://unpkg.com/browse/@angular/flex-layout@9.0.0-beta.28/esm2015/core.js shows that the published NPM files contain `StyleUtils.ɵprov0` instead of `StyleUtils.ɵprov` (See `ɵprov0` instead of `ɵprov`). 

What is the process by which you are creating this package so that it contains suffixed symbols?

**Comment by CaerusKaru** [comment]
We're looking into it. The process right now is a combination of `ngc` and `rollup`. Likely there's a case we need to cover in our custom handling of the resources. I'll consult with @devversion tomorrow, and cut a new release in time for v9 final.

**Comment by devversion** [comment]
Great find @mhevery. @CaerusKaru Here is a PR that fixes the symbol renaming: #1191. I think since we don't have any tests for Ivy in this repository, we should double-check if this PR solves the overall issue.

**Comment by CaerusKaru** [comment]
I confirmed that this issue is fixed. I'm going to wait until the next release is published (hopefully tonight) before closing out this issue.

**Comment by CaerusKaru** [comment]
This should be fixed by v9.0.0-beta.29, now available on NPM.
